paper/scripts/README.md

```diff
@@ -0,0 +1,103 @@
+# Scripts 
+All the scripts in this directory assume that the trained models is supposed to **contain the information needed** like the dataset, the number of training samples, the modality, etc. This information are automatically stored if you use our training script. Otherwise you have to manually modify the lines about the checkpoint**
```

Comment:
is -> are

Reply:
fixed in [9a416df](https://github.com/KRLGroup/memory-wrap/pull/3/commits/9a416dfcc363a1ca4261c4b3a9076b471845d140)